package.json

```diff
@@ -71,5 +81,7 @@
 		"typescript": "^4.0.2",
 		"vscode-test": "^1.4.0"
 	},
-	"dependencies": {}
+	"dependencies": {
+		"html2svg": "^1.0.1"
```

Comment:
You didn't import the html2svg library, this dependency is redundant. Please remove it.